enhancements/sig-architecture/32-extensiblescheduling/32-extensiblescheduling.md

```diff
@@ -110,6 +111,10 @@ type AddOnPlacementScoreItem struct {
 	// +kubebuilder:validation:Maximum:=100
 	// +required
 	Value int32 `json:"value"`
+
+	// Quantity defines the original value of the score.
+	// +optional
+	Quantity resource.Quantity `json:"quantity"`
```

Comment:
this could be quite noisy, brings frequent update on the API. We would need to consider how this should be rate limited.

Reply:
Yes, good suggestion. Add more explanation for the new field:  
1. It should be updated together with the value of the score. 
2. For the resources that are updated frequently, suggest to update at a reasonable interval, eg, 60s. 
3. An expired score will be treated as value 0 and quantity 0 in the placement controller.